### DISCUSSION TITLE
用ALIST列表的链接无法通过其它网站调用。

### DISCUSSION THREAD

**Discussion by fzlychee2019** [AlistGo/alist#1005] [General]
在其他网站中调用ALIST的图片和视频链接，无法显示，用onemanager列表出来的图片和视频是可以在其它网站中调用的。ALIST是否可以增加对图片和视频的调用功能？这样就可以把列表功能做成网站建设的附件库来使用了。目前这块Onemanager做得挺好的，只是它支持的云盘比较少，只支持onedrive和阿里云盘，没象ALIST支持得这么齐全。

**Comment by xhofe** [comment]
本来就支持。

**Comment by fzlychee2019** [comment]
象这样一张图片，在ALIST中是可以正常显示的。
![image](https://user-images.githubusercontent.com/61218953/165578805-f5b0b24c-0691-424d-aadd-55bcbb5ec01a.png)
把这张图片的链接地址输入到网站链接中去，
![image](https://user-images.githubusercontent.com/61218953/165578948-a21dd57e-3c28-4d9a-9f7c-0b6d16821a24.png)
在网站中就无法显示这个链接过来的图片了，但是用onemanager链接的图片就可以正常显示。
![image](https://user-images.githubusercontent.com/61218953/165579121-45d026a2-edc1-4755-bd95-b22429cea1df.png)

**Comment by fzlychee2019** [comment]
同样的图片，从onemanager中链接过来，一切显示正常。
![image](https://user-images.githubusercontent.com/61218953/165579670-4c0f6775-4a6e-499b-b197-a53d38aa1267.png)
![image](https://user-images.githubusercontent.com/61218953/165579809-3653ddc5-5eb4-4baf-8ada-d7c9e7b26bb1.png)
![image](https://user-images.githubusercontent.com/61218953/165579914-b6ab858d-5dfc-4a51-9125-11a87647d593.png)

**Comment by fzlychee2019** [comment]
找到原因了，直接复制过来的链接地址是不行的，在域名后得加上/d/才可以。
象刚才图片的地址为http://test.xmssjt.com.cn:5244/xmssjt/xmssjt/www/uploads/images/20190730/1_2019073010323333855.jpeg
必须在域名后增加d/才可以，变成http://test.xmssjt.com.cn:5244/d/xmssjt/xmssjt/www/uploads/images/20190730/1_2019073010323333855.jpeg，就是目录得增加一个d目录，才是可以显示的真实地址。

**Reply by NovaShen0724** [reply]
无法通过您的方法访问
https://nas.novashen.top/d/%E4%B8%AA%E4%BA%BA/%E6%B1%9F%E7%BB%B5%E6%81%92%E7%9A%84%E7%94%B5%E4%BF%A1%E7%8E%8B%E5%9B%BD_%E7%8E%8B%E5%B0%9A.pdf
报错401，可否答疑谢谢！alist v3

**Comment by ruoli6** [comment]
新版alist v3能关闭下载链接所需的签名吗
